app/feature/feature-payments/src/main/kotlin/com/hedvig/android/feature/payments/data/PaymentConnection.kt

```diff
@@ -3,23 +3,19 @@ package com.hedvig.android.feature.payments.data
 import kotlinx.serialization.Serializable
 
 @Serializable
-internal data class PaymentConnection(
-  val connectionInfo: ConnectionInfo?,
-  val status: PaymentConnectionStatus,
-) {
+internal sealed interface PaymentConnection {
```

Comment:
Cool

Reply:
Indeed, I was pulling my hair (the little I got) when I couldn't understand how this model had null content on an active payment connection 😅